ethereum/test/MessageQueue.t.sol

```diff
@@ -35,34 +35,34 @@ contract MessageQueueTest is TestHelper {
 
     function setUp() public override {
```

Comment:
Can't you use `setUp` from `TestHelper.t.sol`?

Reply:
done